### PR TITLE
refactor tensorflow inception

### PR DESCRIPTION
**Pull request by cheyang**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: incubator/tensorflow-inception/values.yaml [hunk 1]
```diff
@@ -8,9 +8,10 @@ servicePort: 9090
 containerPort: 9090
 component: "tensorflow-inception"
 replicas: 1
-image: "quay.io/lachie83/inception_serving"
-imageTag: "latest"
-imagePullPolicy: "IfNotPresent"
+image:
+  repository: "quay.io/lachie83/inception_serving"
+  tag: "latest"
```
Comment (unguiculus) [inline incubator/tensorflow-inception/values.yaml:13]: Please use a stable release.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/tensorflow-inception/values.yaml (hunk 1 above)
Comment (cheyang) [reply incubator/tensorflow-inception/values.yaml:13]: What do you mean a stable release? I notice this image only has latest tag. See https://quay.io/repository/lachie83/inception_serving?tag=latest&tab=tags

Comment (unguiculus) [reply incubator/tensorflow-inception/values.yaml:13]: In general, we require stable versions. Is there another image you could use?

@lachie83 Care to provide a stable image?

Comment (thomasjungblut) [reply incubator/tensorflow-inception/values.yaml:13]: @cheyang I'm currently fixing the GPU support for this chart and updating tensorflow serving. I created the repo with corresponding quay build: https://github.com/thomasjungblut/tfs-inception.

Can I attach the changes in here? I'll also provide a stable tag ;-)

@lachie83 I hope you don't have an issue with replacing your version of the image, couldn't find any dockerfile for it?

Comment (thomasjungblut) [reply incubator/tensorflow-inception/values.yaml:13]: created https://github.com/kubernetes/charts/pull/6698

Comment (cheyang) [reply incubator/tensorflow-inception/values.yaml:13]: Sure, I think I can close my fix. Thanks.

**Comment by unguiculus** [comment]
Closing in favor of #6698

**Pull request closed without merging**